docs/theming-flatpak.rst

```diff
@@ -0,0 +1,46 @@
+Theming Flatpak
+===============
+
+Flatpak applications cannot directly use the system theme. This happens because all flatpaks are sandboxed and do not have the ability to use data files or libraries in ``/usr`` (where system themes are typically located). This is done so that the format is portable across a wide range of distributions in a reproducible environment, but has the downside of not being able to pick up system themes. The solution to this was to package GTK themes as Flatpaks, as relying upon the host to have the correct version for every flatpak defeats the portability benefits it provides. These themes are provided as `extensions <https://github.com/flatpak/flatpak/wiki/Extensions>`_ to the Gnome runtime.
+
+Installing themes
+-----------------
+
+Instructions for Gtk
+````````````````````
+
+The current Gtk themes are packaged in the `flathub <https://flathub.org/>`_ repository which you can add (if it's not already added) by running::
+
+  $ flatpak remote-add flathub https://flathub.org/repo/flathub.flatpakrepo
+
+To see a list of currently packaged themes you can use the command ``flatpak search gtk3theme`` (available since Flatpak version 0.10.1). In case you use an older version of Flatpak than that (something discouraged by the developers - it is suggested that you use a PPA in order to have a recent version of Flatpak), you can use the command ``flatpak remote-ls flathub | grep org.gtk.Gtk3theme``.
+
+You can install them with ``flatpak install flathub org.gtk.Gtk3theme.Foo``.
+
+Instructions for Qt
+```````````````````
+
+For the Qt theming to work, the flatpak packages kstyle and platformtheme must be installed. These are packed in the kdeapps repository which you can add by running::
+
+  $ flatpak remote-add kdeapps https://distribute.kde.org/kdeapps.flatpakrepo
+
+Afterwards the two packages can be installed with the following commands::
+  
+  $ flatpak install kdeapps org.kde.KStyle.Adwaita//5.9
+  $ flatpak install kdeapps org.kde.PlatformTheme.QGnomePlatform//5.9
+
+Applying themes
+---------------
+
+There is no way to specify the theme Flatpak applications use. The applications will try to match the system theme currently being used, if it corresponds to any of the Flatpak themes installed, and will fall back to Adwaita (if they use Gtk2 or Gtk3) or the default Qt theme (if they use Qt) if a corresponding theme isn't detected. This happens because the ``flatpak`` tool doesn't depend upon Gtk so it will not automatically detect your theme. As of Flatpak 0.10.1, the Flatpak system can detect whether the system themes available correspond to any Flatpak themes available in the repositories, and, if so, will automatically install found themes at update time based upon various DConf values. If the corresponding system themes are not currently being used however, the applications will still fall back to Adwaita or the default Qt theme.
```

Comment:
> There is no way to specify the theme Flatpak applications use.

To be pedantic lets say: There is no ideal way to specify the theme.

> This happens because the ``flatpak`` tool doesn't depend upon Gtk so it will not automatically detect your theme. 

Lets just remove this line now.